docs/indexing.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Indexing
+sidebar_label: Indexing
+slug: /neo4j-slow-query-performance-issue
+---
+
+Ahh, indexing! One of the most common pitfalls in database performance in general, Neo4j included, is the usage of indexes. At first, it may seem what I need and index for in the Graph database, but graphs are abstractions over some other structure you can index and optimize. The wrong usage of the index can make a database unusable, which can be quite painful. Let's see a few common issues with Neo4j database performance and indexes. 
```

Comment:
_At first, it may seem what I need and index for in the Graph database, but graphs are abstractions over some other structure you can index and optimize._  - a bit unclear sentence

Reply:
Refactor it a bit.